models/comments.model.js

```diff
@@ -40,3 +40,17 @@ exports.addComment = (newComment, article_id) =>{
         }
     })
 }
+
+exports.removeComment = (comment_id)=>{
+    return db
+    .query(
+        `DELETE FROM comments
+        WHERE comment_id = $1
```

Comment:
Great use of the parameterized variable here to prevent any SQL injection attacks 👍